tests/test_examples.py

```diff
@@ -838,7 +840,53 @@ def test_var_spool_cwl_checker3() -> None:
 
 
 def test_print_dot() -> None:
-    assert main(["--print-dot", get_data("tests/wf/revsort.cwl")]) == 0
+    cwl_path = get_data("tests/wf/revsort.cwl")
+    cwl_posix_path = Path(cwl_path).as_posix()
+    expected_dot = pydot.graph_from_dot_data(f"""
```

Comment:
Can't use an `f` string in Python 3.5; sorry!

Reply:
as you understood I am not familiar with python3.5 and I realized that I use everything that does not exist in python3.5... 😛 
But the real issue I am looking at is on windows when I am comparing in the test the dot graphs, because it writes paths differently, and I am looking to find access to a windows machine to see how it works to fix it.

Reply:
Yeah, it isn't a problem for me; I understand!

About the paths, it might be better to not output absolute paths in the ".dot" when relative paths are given to `cwltool`. Worse case you do a search and replace at the very end. Might help with the windows issue too

Reply:
So that the `file:///` URIs are still valid you could do `file:///fakepath/cwltool/` and then the relative parts

Reply:
Actually I just fixed it. I parse it and I am just comparing that I have the correct edges in the dot graph by comparing only the fragment of the URI